pkg/plugin/vector/store/qdrant.go

```diff
@@ -0,0 +1,78 @@
+package store
+
+import (
+	"context"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
+	qdrant "github.com/qdrant/go-client/qdrant"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type qdrantSettings struct {
+	Address string
+}
+
+type qdrantStore struct {
+	conn              *grpc.ClientConn
+	collectionsClient qdrant.CollectionsClient
+	pointsClient      qdrant.PointsClient
+}
+
+func newQdrantStore(s qdrantSettings) (ReadVectorStore, func(), error) {
+	conn, err := grpc.DialContext(context.Background(), s.Address, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
A TODO or issue would probably be ok for now, but we should add some auth settings. I don't love reading "insecure" in a PR :D

Reply:
Good call, I added some TLS and auth settings following [the qdrant example](https://github.com/qdrant/go-client/blob/master/examples/authentication/main.go).